notebooks/official/migration/sdk-hyperparameter-tuning.ipynb

```diff
@@ -82,12 +83,12 @@
         "\n",
         "This tutorial uses the following Google Cloud ML services:\n",
         "\n",
-        "- `Vertex AI Training`\n",
-        "- `Vertex AI Hyperparameter Tuning`\n",
+        "- Vertex AI Training\n",
+        "- Vertex AI Hyperparameter Tuning\n",
```

Comment:
change: "Vertex AI Hyperparameter Tuning"
to: "Vertex AI hyperparameter tuning"